core/storage/skill/gae_models.py

```diff
@@ -197,7 +197,7 @@ def get_unpublished_by_creator_id(cls, user_id):
         """
         return cls.query(
             cls.creator_id == user_id,
-            cls.skill_is_private == True) #pylint: disable
+            cls.skill_is_private == True) #pylint: disable=singleton-comparison
```

Comment:
No, please remove these pragmas -- such comparisons should not be made (See [PEP8 standard](https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_true.html#comparing-things-to-true-the-wrong-way) for this). This can be changed to `cls.skill_is_private` only. Ditto below!
Thanks!

Reply:
If I do that, then I get this error, do you know another way around it?
```TypeError: Cannot filter a non-Node argument; received BooleanProperty('skill_is_private', required=True, default=True)```

Reply:
Will this work?
`cls.skill_is_private.IN([True])`

Reply:
This is actually one of the rare cases where a pylint pragma is justified -- the GAE query syntax is quite specific. I think using a direct equality is better; we do that elsewhere.